### PR TITLE
Fix typo in AMM ledger_entry API example

### PR DESCRIPTION
**Pull request by JST5000**
The `amm_info` request was missing a comma - this adds it back :) 

(Confirmed this new transaction was well-formed using the Websocket API tool)

image

### CODE REVIEW

**Comment by JST5000** [comment]
(Feel free to just merge this in if it looks good and has enough approvals - I may be slow to respond since I don't have notifications for this repo)

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**
